lambda/src/main/scala/pricemigrationengine/model/Subscription.scala

```diff
@@ -0,0 +1,3 @@
+package pricemigrationengine.model
+
+case class Subscription(subscriptionNumber: String)
```

Comment:
Yes I guess so. 

The only thing is that the processingStage of each put/update is set in the serialiser for the case class used in that update/put.

Ie the case class is tied to a specific processingStage.

The CohortItem is used for the reads and has optional fields for everything other than subscription number.

In retrospect this doesn't actually work all that well, maybe we should just have the CohortItem case class and explicitly set the optional fields and the processing stage as each handler requires.

Thats probably worth splitting out into a separate refactoring/PR.

It shouldnt take long and I can do it as part of this ticket.

Reply:
Hum, maybe the change I suggested is a bit more involved that I expected.

Ill implement your suggestions and we can decide on whether to do the refactor if there is time.

Reply:
I think it would be a good idea if we refactored so that the `CohortTable` only knows how to read and write `CohortItems`.  Then convert `EstimationResult` etc into `CohortItem` before writing back to the CohortTable.  But that's one for another PR.